src/ui/marker.js

```diff
@@ -334,6 +334,7 @@ export default class Marker extends Evented {
             if (this._lngLat) this._popup.setLngLat(this._lngLat);
 
             this._element.setAttribute('role', 'button');
+            if (!this._element.hasAttribute('aria-label')) this._element.setAttribute('aria-label', 'Map marker');
```

Comment:
Since we're setting this attribute for the default marker in the constructor regardless of whether it's interactive or not, should we move this line to the constructor "else" block for consistency?